### PR TITLE
upgrade to libbcc 0.9.0

### LINKED ISSUE #49: Upgrade to libbcc 0.9.0
Current Dockerfile does not build since libbcc has been updated to 0.9.0 upstream.

I propose updating ebpf_exporter to use 0.9.0.
```
ubuntu@ip-100-66-117-245:~/ebpf_exporter$ git checkout upstream/master 
ubuntu@ip-100-66-117-245:~/ebpf_exporter$ make
..
Step 2/4 : RUN apt-get update &&
     apt-get install -y apt-transport-https &&
     apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 648A4A16A23015EEF4A66B8E4052245BD4284CDD &&
     echo "deb https://repo.iovisor.org/apt/xenial xenial main" > /etc/apt/sources.list.d/iovisor.list &&
     apt-get update &&
     apt-get install -y libbcc=0.8.0-1 linux-headers-amd64
..
E: Version '0.8.0-1' for 'libbcc' was not found
```

### PR DESCRIPTION
**Pull request by jasonk000**
Upgrades libbcc to 0.9.0, fixes #49

### CODE REVIEW

File: exporter/attach.go [hunk 1]
```diff
@@ -79,14 +80,20 @@ func attachSomething(module *bcc.Module, loader probeLoader, attacher probeAttac
 	return tags, nil
 }
 
+func withMaxActive(attacherWithMaxActive probeAttacherWithMaxActive, maxActive int) probeAttacher {
```
Comment (bobrik) [inline exporter/attach.go]: A comment would be nice

Comment (jasonk000) [reply exporter/attach.go]: added in 0e2281b

File: Dockerfile [hunk 2]
```diff
@@ -1,12 +1,12 @@
-FROM golang:1.11.4-stretch
```
[the remark this answers is not in the document]
Comment (jasonk000) [reply Dockerfile]: done in 3dc71e8

**Comment by bobrik** [comment]
Thanks!

**Pull request merged**